archlinuxcn-mirrorlist

```diff
@@ -1,112 +1,128 @@
+## 吉林大学 (吉林长春) (ipv4, ipv6)
 # Server = https://mirrors.jlu.edu.cn/archlinuxcn/$arch
 
-## 浙江大学 (浙江杭州) (ipv4, ipv6, http, https)
+## 浙江大学 (浙江杭州) (ipv4, ipv6)
 # Server = https://mirrors.zju.edu.cn/archlinuxcn/$arch
 
-## 重庆大学 (重庆) (ipv4, ipv6, http, https)
+## 重庆大学 (重庆) (ipv4, ipv6)
 # Server = https://mirrors.cqu.edu.cn/archlinuxcn/$arch
 
-## SJTUG 软件源镜像服务 (上海) (ipv4, ipv6, https)
+## 重庆邮电大学 (重庆) (ipv4, ipv6)
+# Server = https://mirrors.cqupt.edu.cn/archlinuxcn/$arch
+
+## 上海交通大学 (上海) (ipv4, ipv6)
 # Server = https://mirror.sjtu.edu.cn/archlinux-cn/$arch
 
-## 南京大学 (江苏南京) (ipv4, ipv6, http, https)
+## 南京大学 (江苏南京) (ipv4, ipv6)
 # Server = https://mirrors.nju.edu.cn/archlinuxcn/$arch
 
-## 南方科技大学 (广东深圳) (ipv4, ipv6, http, https)
+## 南方科技大学 (广东深圳) (ipv4, ipv6)
 # Server = https://mirrors.sustech.edu.cn/archlinuxcn/$arch
 
-## 武昌首义学院 (湖北武汉) (ipv4, ipv6, http, https)
+## 武昌首义学院 (湖北武汉) (ipv4, ipv6)
 # Server = https://mirrors.wsyu.edu.cn/archlinuxcn/$arch
 
-## 北京交通大学 (北京) (ipv4, ipv6, https)
+## 北京交通大学 (北京) (ipv4, ipv6)
 # Server = https://mirror.bjtu.edu.cn/archlinuxcn/$arch
 
-## 兰州大学 (甘肃兰州) (ipv4, ipv6, http, https)
-# Server = https://mirror.lzu.edu.cn/archlinuxcn/$arch
+## 兰州大学 (甘肃兰州) (ipv4, ipv6)
+# Server = https://mirrors.lzu.edu.cn/archlinuxcn/$arch
 
-## 西安交通大学 (陕西西安) (ipv4, ipv6, http, https)
+## 西安交通大学 (陕西西安) (ipv4, ipv6)
 # Server = https://mirrors.xjtu.edu.cn/archlinuxcn/$arch
 
-## 南阳理工学院 (河南南阳) (ipv4, ipv6, http, https)
+## 南阳理工学院 (河南南阳) (ipv4, ipv6)
 # Server = https://mirror.nyist.edu.cn/archlinuxcn/$arch
 
-## 南京工业大学 (江苏南京) (ipv4, ipv6, http, https)
+## 南京工业大学 (江苏南京) (ipv4, ipv6)
 # Server = https://mirrors.njtech.edu.cn/archlinuxcn/$arch
 
-## 上海科技大学 (上海) (ipv4, https)
+## 上海科技大学 (上海) (ipv4)
 # Server = https://mirrors.shanghaitech.edu.cn/archlinuxcn/$arch
 
-## 中国科学院软件研究所 (北京) (ipv4, https)
+## 中国科学院软件研究所 (北京) (ipv4)
 # Server = https://mirror.iscas.ac.cn/archlinuxcn/$arch
 
-## 北京邮电大学 (北京) (ipv4, ipv6, http, https)
+## 北京邮电大学 (北京) (ipv4, ipv6)
 # Server = https://mirrors.bupt.edu.cn/archlinuxcn/$arch
 
-## NCKU CCNS (Taiwan) (ipv4, http, https)
+## 國立成功大學 (臺灣臺南) (ipv4)
 # Server = https://archlinux.ccns.ncku.edu.tw/archlinuxcn/$arch
 
-## xTom (Hong Kong server) (Hong Kong) (ipv4, ipv6, http, https)
+## xTom (香港伺服器) (香港) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.hk/archlinuxcn/$arch
 
-## xTom (US server) (San Jose, CA, United States) (ipv4, ipv6, http, https)
+## xTom (United States server) (San Jose, United States) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.us/archlinuxcn/$arch
 
-## xTom (Netherlands server) (Amsterdam, the Netherlands) (ipv4, ipv6, http, https)
+## xTom (Netherlands server) (Amsterdam, Netherlands) (ipv4, ipv6)
 # Server = https://mirrors.xtom.nl/archlinuxcn/$arch
 
-## xTom (Germany server) (Duesseldorf, Germany) (ipv4, ipv6, http, https)
+## xTom (Germany server) (Dusseldorf, Germany) (ipv4, ipv6)
```

Comment:
"Duesseldorf" is the ASCII transscript of "Düsseldorf". Maybe we should just use "Düsseldorf".